packages/block-library/src/embed/block.json

```diff
@@ -43,12 +43,29 @@
 	"supports": {
 		"align": true,
 		"spacing": {
-			"margin": true
+			"margin": true,
+			"padding": true
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
+				"color": true,
+				"width": true,
+				"style": true
+			},
+			"__experimentalSkipSerialization": true
 		}
 	},
+	"selectors": {
+		"border": ".wp-block-embed__wrapper "
```

Comment:
The convention with these selectors is to explicitly include the block's class. In this case, `.wp-block-embed`. 

I'm not 100% whether it is required but it does make it clearer the intended target for the styles. That block class is also tweaked when generating block style variation selectors for the block. 

One last nit, there's an unnecessary space at the end of that selector. Better safe than sorry, we should clean that up in case theres some theme.json processing somewhere that wasn't expecting it.